source/_docs/installation/virtualenv.markdown

```diff
@@ -34,20 +32,15 @@ It is recommended to use the [advanced guide](/docs/installation/raspberry-pi/)
     ```
     $ source bin/activate
     ```
- 4. Install wheel:
-    ```
-    $ python3 -m pip install wheel
-    ```
- 5. Install Home Assistant:
+ 4. Install Home Assistant:
     ```
     $ python3 -m pip install homeassistant
     ```    
- 6. Configure it to [autostart](/docs/autostart/)
- 7. Or run Home Assistant manually:
+ 5. Run Home Assistant:
     ```
     $ hass --open-ui
     ```
- 8. You can now reach the web interface on `http://ipaddress:8123/` - the first start may take up to 20 minutes before the web interface is available
+ 6. You can now reach the web interface on `http://ipaddress:8123/` - the first start may take a couple of minutes before the web interface is available
```

Comment:
How about a note that on lower end hardware, such as a Pi, it may take longer. I have seen startups of 10 to 20 minutes on a Pi often enough.

Reply:
```suggestion
 6. You can now reach the web interface on `http://ipaddress:8123/` - the first start may take some time before the web interface is available, especially on lower-end hardware, sure as a Pi. 
```

Reply:
I guess it could depend on if compiled binaries are available for the platform that it is being installed upon. 

I do still want to give it an estimated number Frenck, "some time" can mean too many things. I think I will stick to "a couple of minutes" but add your sentence about lower-end hardware.